examples/integration_test.py

```diff
@@ -359,9 +360,13 @@ def verify_stats_cb():
     """ Verify stats_cb """
 
     def stats_cb(stats_json_str):
+        global good_stats_cb_result
         stats_json = json.loads(stats_json_str)
         if 'test' in stats_json['topics']:
-            print("# app_offset stats for topic test partition 0: %d" % stats_json['topics']['test']['partitions']['0']['app_offset'])
+            app_offset = stats_json['topics']['test']['partitions']['0']['app_offset']
```

Comment:
probably want to wrap this in a try: if any of the sub-dict keys are missing and just print the json blob on failure.